test/unit/test_arguments.py

```diff
@@ -114,7 +115,52 @@ def test_device_args(test_params, expected_params):
               conv_embed_num_highway_layers=4,
               conv_embed_pool_stride=5,
               conv_embed_add_positional_encodings=False,
-              rnn_attention_in_upper_layers=False))])
+              rnn_attention_in_upper_layers=False)),
+    ('--rnn-scale-dot-attention', dict(params=None,
```

Comment:
can we remove this test? We stripped down all of these test_arguments tests to the default case ('') to not have people have to add arguments all over the place.

Reply:
done